Firestore/Example/Tests/Integration/FSTTransactionTests.mm

```diff
@@ -117,7 +117,7 @@ - (void)testGetNonexistentDocumentThenFailPatch {
         XCTAssertEqualObjects(error.domain, FIRFirestoreErrorDomain);
         // TODO(dimond): This is probably the wrong error code, but it's what we use today. We
         // should update the code once the underlying error was fixed.
-        XCTAssertEqual(error.code, FIRFirestoreErrorCodeFailedPrecondition);
+        XCTAssertEqual(error.code, FIRFirestoreErrorCodeAborted);
```

Comment:
This was never the right error code. We should change this error to match the one in your spreadsheet.

Here and throughout.